hubstorage/serialization.py

```diff
@@ -1,4 +1,5 @@
 from json import dumps, loads
+from msgpack import Unpacker
```

Comment:
I prefer to be deterministic and require msgpack, we also bump the version accordingly to the new requirement.
Although  my worry with msgpack are those PaaS that doesn't like non-native python packages like GAE but I don't know if it is still the case.